pages/content/pixi/index.md

```diff
@@ -0,0 +1,15 @@
+---
+$sitemap:
+  enabled: false
+globalText:
+  analyzeHeadline: Analyze your AMP page
+  analyzeFieldPlaceholder: Enter your URL
+  analyzeButton: Analyze
+tags:
```

Comment:
Is tags the right name here? Wouldn't these be used for headlines as well?

Reply:
Moved the tags into the "staticText" since here they are only the translation for the tag IDs.